DuckDuckGo/SettingsViewController.swift

```diff
@@ -113,7 +113,7 @@ class SettingsViewController: UITableViewController {
     }
 
     private var shouldShowSyncCell: Bool {
-        return syncService.featureFlags.contains(.userInterface) || internalUserDecider.isInternalUser
```

Comment:
the `|| internalUserDecider.isInternalUser` is a remainder from the original feature flags PR when we were supposed to show the UI to internal users anyway, even if Sync top-level feature flag is disabled. We've replaced that on Apple platforms by setting top-level flag to "internal", so this code shouldn't end up here.